backend/tests/userTests.test.js

```diff
@@ -0,0 +1,89 @@
+// M6 Q3
+//import {jest} from '@jest/globals';
+//import axios from 'axios';
+var axios = require('axios');
+const { ObjectID } = require('bson');
```

Comment:
Codacy has a fix for the issue: [Replace `'bson'` with `"bson"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
const { ObjectID } = require("bson");
```